build.gradle.kts

```diff
@@ -76,10 +77,13 @@ kotlin {
             val junitVersion: String by project
             val jctoolsVersion: String by project
             val mockkVersion: String by project
+            val gradleToolingApiVersion: String by project
             dependencies {
                 implementation("junit:junit:$junitVersion")
                 implementation("org.jctools:jctools-core:$jctoolsVersion")
                 implementation("io.mockk:mockk:${mockkVersion}")
+                implementation("org.gradle:gradle-tooling-api:${gradleToolingApiVersion}")
+                runtimeOnly("org.slf4j:slf4j-simple:1.7.10")
```

Comment:
Why do you need it?

Reply:
Do you mean why we need `slf4j`? We need it for Gradle API. Otherwise there is the following warning in the console `SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder"...`. I can drop it if you are OK with that.

Reply:
BTW, do you need to add these dependencies for jvmTest, or is adding them only for your new task sufficient?

I'm fine with adding the dependency, but please comment in the code on why it is necessary.

Reply:
> BTW, do you need to add these dependencies for jvmTest, or is adding them only for your new task sufficient?

It is enough to add them only for my tests. Should I create a new source set for trace debugger tests?